client/i18n.js

```diff
@@ -7,7 +7,7 @@ const fallbackLng = ['en-US'];
 const availableLanguages = ['en-US', 'es-419'];
 
 const options = {
-  loadPath: '/translations/{{lng}}/translations.json',
```

Comment:
I think this should stay as `'/translations/{{lng}}/translations.json'`, but the static file config in `server/server.js` needs to change...